src/pages/index.md

```diff
@@ -1,70 +1,26 @@
-    Kaldi is the coffee store for everyone who believes that great coffee
-    shouldn't just taste good, it should do good too. We source all of our beans
-    directly from small scale sustainable farmers and make sure part of the
-    profits are reinvested in their communities.
+    We go further than just the technical skills that our students deserve. Our students receive supplementary skills in College Prep, Financial Literacy, and Entrepreneurship. These vital skills are taught by our program to amplify our impact to maximize success.
 description: >-
-  Kaldi is the ultimate spot for coffee lovers who want to learn about their
-  java’s origin and support the farmers that grew it. We take coffee production,
-  roasting and brewing seriously and we’re glad to pass that knowledge to
-  anyone.
+  Giving opportunities that inspire young people to explore, discover, and create. By using the talent that we see in underrepresented students, we are able to make a real impact in their lives and watch as they go on to shape their communities. By starting at the source, we can organically grow a culture of ambition and success. The triumphs of our students goes on to be reflected in making the technology world more diverse and bringing the voices that are so often unheard to the forefront.
 intro:
   blurbs:
     - image: /img/coffee.png
       text: >
-        We sell green and roasted coffee beans that are sourced directly from
-        independent farmers and farm cooperatives. We’re proud to offer a
-        variety of coffee beans grown with great care for the environment and
-        local communities. Check our post or contact us directly for current
-        availability.
+        Pi515 successfully creates a safe space where students from a wide array of backgrounds, socio-economic and social status, and nationalities come together to learn from our program and each other. Individually, they learn to build skills and self-confidence. Together, they learn to build community – and the community is part of what keeps them invested in being part of the program. We provide the program at no cost to their families and give them a laptop when they graduate.
     - image: /img/coffee-gear.png
       text: >
-        We offer a small, but carefully curated selection of brewing gear and
-        tools for every taste and experience level. No matter if you roast your
-        own beans or just bought your first french press, you’ll find a gadget
-        to fall in love with in our shop.
+        We provide equipment directly where it is needed at the time that it makes the most impact. Professionals in tech come in to teach the high school level who in turn give back immediately by mentoring their younger peers. With small classes, we give the one-on-one attention that these students need. We hire high school students to give them their first opportunity in industry, giving them a foot in the door and a strong resume as soon as possible.
```

Comment:
>high school
->
>high-school

Reply:
Also, "giving them a foot in the door" reads weird. What about something like "allowing them to get their foot in the door"?

Reply:
I think High School is correct, non-hyphenated. 
Changed the foot door wording